src/consensus/aft/impl/view_change_tracker.h

```diff
@@ -239,7 +240,7 @@ namespace aft
       }
 
       auto& vc = it->second;
-      ccf::ViewChangeConfirmation nv(vc.view);
+      ccf::ViewChangeConfirmation nv(vc.view, from);
```

Comment:
Is `from` the node id that issued the view change confirmation? Is that effectively the primary at the time?

Reply:
can we rename that to primary?